src/web/mjs/connectors/NewToki.mjs

```diff
@@ -1,99 +1,44 @@
-import Connector from '../engine/Connector.mjs';
-import Manga from '../engine/Manga.mjs';
+import GnuBoard5BootstrapBasic2 from './templates/GnuBoard5BootstrapBasic2.mjs';
 
-export default class NewToki extends Connector {
+export default class NewToki extends GnuBoard5BootstrapBasic2 {
 
     constructor() {
         super();
         super.id = 'newtoki';
         super.label = 'NewToki';
         this.tags = [ 'manga', 'webtoon', 'korean' ];
         this.url = 'https://newtoki.com';
```

Comment:
URL cannot be reached, currently it is https://newtoki83.com
Consider using the same getter/setter mechanism to store the URL during `initiqlizeConnector` in settings as it is done in [CopyToon](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/CopyToon.mjs).